financial-connections/src/main/java/com/stripe/android/financialconnections/features/accountpicker/AccountPickerScreen.kt

```diff
@@ -370,89 +355,6 @@ private fun FinancialConnectionRadioButton(
     }
 }
 
-@Composable
-@Suppress("MagicNumber")
-private fun AccountItem(
```

Comment:
Moved this to its own file to make it reusable across Panes. Also realized `PartnerAccountUI` was redundant, so removed it and just used the raw account class.